exercises/practice/ledger/.docs/instructions.append.md

```diff
@@ -0,0 +1,4 @@
+# Instructions append
+
+When working with the ledger, treat the `e` symbol as a stand-in for the Euro currency symbol (€).
+This substitution ensures that the program remains functional and adheres to the ASCII-only constraint, without sacrificing usability.
```

Comment:
```suggestion
Because Cairo only supports ASCII characters, and the Euro currency symbol (€) is _not_ an ASCII character, we will be using the character `e` as a substitute.
```
Simpler